partner_feeds/ge_tasks.py

```diff
@@ -103,9 +103,9 @@ def update_posts_for_feed_task(partner):
                 p.url = entry.link
 
                 # try to get the date of the entry, otherwise, use the current date
-                if hasattr(entry, 'published_parsed'):
+                if hasattr(entry, 'published_parsed') and entry.published_parsed is not None:
```

Comment:
Would this slightly more concise way also work? `if getattr(entry, 'published_parsed', None):`